### ISSUE TITLE
Fresh install and no charts listed....

### ISSUE THREAD

**Issue by natiki** [kubeapps/kubeapps#472]
Hi,

I am running a Minikube cluster on Windows and have done all the steps of getting started including generating my token.

After running `kubeapps dashboard` I go to the Charts tab and no charts are displayed. Looking at DevTools I see a bad gateway:

![image](https://user-images.githubusercontent.com/4738140/43999238-d5bdd568-9e4b-11e8-9102-902e3da4d98b.png)

There are repositories configured:

![image](https://user-images.githubusercontent.com/4738140/43999248-15fe45ea-9e4c-11e8-8502-b1a90b4ca277.png)

1) So what am I missing here? 
2) Is there a way to not have to enter the dashboard token each time?

**Comment by andresmgot** [comment]
Hi @natiki, 

Can you check the logs of the chartsvc pod? It should show more info about the issue:

```
kubectl logs -n kubeapps kubeapps-chartsvc-...
```

Also, are the pods in the `kubeapps` healthy? A possible cause can be that the Mongodb database is not properly populated. You can check that executing:

```
kubectl get pods -n kubeapps
```

In any case, we released the past week the version v1.0.0-alpha.5 of Kubeapps (that you can install with Helm). Would you mind to clean up your current installation and try that one?

**Comment by natiki** [comment]
- How can I find the version I am currently running?

**Comment by andresmgot** [comment]
If you run `kubeapps version` it will tell which version you are running. This command line tool is deprecated though. If you install it with the Helm chart you will get the latest version available (v1.0.0-alpha.5).

**Comment by natiki** [comment]
I will install alpha-5 and let you know. Below is the debug info for posterity. Seems the MongoDB is the issue:

```
C:\Kubernetes>kubeapps version
Kubeapps Installer version: v1.0.0-alpha.4

C:\Kubernetes>kubectl get pods -n kubeapps
NAME                                        READY     STATUS             RESTARTS   AGE
apprepo-sync-incubator-1534050000-m8n6s     1/1       Running            0          1d
apprepo-sync-incubator-qjtsh-9lwmd          1/1       Running            0          1d
apprepo-sync-stable-1534050000-t5h5s        1/1       Running            0          1d
apprepo-sync-stable-z75dl-26t55             1/1       Running            0          1d
apprepo-sync-svc-cat-1534050000-2q68n       1/1       Running            0          1d
apprepo-sync-svc-cat-v98cs-gfr7z            1/1       Running            0          1d
apprepository-controller-558c68c455-7jrsr   1/1       Running            0          1d
chartsvc-6d54b6df96-dzq4h                   1/1       Running            0          1d
kubeapps-594c9d4fb9-58cch                   1/1       Running            5          1d
kubeapps-dashboard-ui-64687fb98c-4842n      1/1       Running            0          1d
mongodb-fdbd4455c-mjclx                     0/1       CrashLoopBackOff   336        1d
tiller-deploy-6c54755696-8zfqb              2/2       Running            3          1d

C:\Kubernetes>kubectl logs -n kubeapps chartsvc-6d54b6df96-dzq4h
```

**Comment by natiki** [comment]
Hi,

I am now trying with alpha5 but running into different issues:

The install failed somewhere so I wanted to do a reinstall:

```
c:\Kubernetes>helm ls --all kubeapps
NAME            REVISION        UPDATED                         STATUS  CHART           NAMESPACE
kubeapps        1               Mon Aug 13 20:27:39 2018        FAILED  kubeapps-0.1.0  kubeapps

c:\Kubernetes>helm del --purge kubeapps
release "kubeapps" deleted
```

However the delete does not seem to take out all the resources required. Leaving a role hanging around:

```
c:\Kubernetes>helm install --name kubeapps --namespace kubeapps bitnami/kubeapps
Error: serviceaccounts "kubeapps-apprepository-jobs-bootstrap" already exists
```

1) So how do I get rid of the role?
2) What logs do you need to show the install failure?

**Comment by natiki** [comment]
@arapulido This now worked perfectly. The steps I followed were:

- Fresh Minikube v0.28.2 on Windows 10
- Followed [getting started](https://github.com/kubeapps/kubeapps/blob/master/docs/user/getting-started.md)
- Installed CockroachDB 2.05 and that worked with a small issue (see below). However I suspect that is a chart issue (I include it here only for completeness) and this case can be closed.

When I installed the Cockroach 2.05 chart I changed the ExternalHttpPort to 8111 and received the following error in the browser console:

![image](https://user-images.githubusercontent.com/4738140/44942619-6f442900-adf9-11e8-9372-d0b153888d4d.png)
